packages/commonwealth/server/database.ts

```diff
@@ -80,6 +80,15 @@ export const sequelize = new Sequelize(DATABASE_URI, {
   },
 });
 
+console.log('Connecting to database...', DATABASE_URI);
+
+sequelize.authenticate().then(() => {
```

Comment:
What does this / why did we need to add this?

Reply:
I added it for debugging, to see if Postgres was correctly connecting and what URI it was using, I can get rid of it, but the function on line 85 is useful to see the result for now